pkg/storaged/details.js

```diff
@@ -63,7 +63,7 @@
             mdraid_add_disk: function mdraid_add_disk(path) {
                 var mdraid = client.mdraids[path];
 
-                dialog.open({ Title: _("Add Disks"),
+                dialog.open({ Title: _("Add Spare Disks"),
```

Comment:
Yeah, hmm.  The distinction I am trying to make is between adding a disk in order to make the RAID device larger, and adding a disk in order to add more redundancy.